libsignal-protocol/src/errors.rs

```diff
@@ -112,6 +114,22 @@ impl InternalErrorCode for i32 {
             Some(e) => Err(e),
         }
     }
+
+    fn into_code(self) -> i32 { self }
+}
+
+impl<T> InternalErrorCode for Result<T, InternalError>
+where
+    T: Any,
```

Comment:
i used `Any` because i just want a general type, is there is a better replacement ? a `Debug` maybe ?

Reply:
You alright, i will edit that and remove the bounda